internal/pkg/fieldtags/testdata/src/tests/test.go

```diff
@@ -15,10 +15,9 @@
 package fieldtags
 
 type Person struct {
-	password, creds      string      `levee:"source"`               // want "tagged field: password, creds"
-	secret               string      `json:"secret" levee:"source"` // want "tagged field: secret"
-	another              interface{} "levee:\"source\""             // want "tagged field: another"
```

Comment:
I think we should still support this case, since `structtag` doesn't complain about it and `StructTag.Get` supports it, e.g. see the example program below. If we use `strconv.Unquote` we can convert the string literal from the `ast` into something `StructTag.Get` can handle, e.g. `"levee:\"source\""` becomes `levee:"source"` with `Unquote`.

Example program showing how StructTag handles the case above (`f.Tag` is a `StructTag`):
```go
package main

import (
	"fmt"
	"reflect"
)

func main() {
	f, _ := reflect.TypeOf(myStruct{}).FieldByName("myField")
	fmt.Println(f.Tag)
}

type myStruct struct {
	myField string "key:\"value\""
}
```
(this prints `key:"value"`)

Reply:
I didn't know about `strconv.Unquote`!  That keeps things nice and tidy then.